src/services/storage/index.ts

```diff
@@ -37,20 +45,50 @@ export class Storage implements StorageDriver {
 			return this.storage._driver
 		}
 		public async get(id: string): Promise<StoredUrl> {
-			return this.driver.url.get(id)
+			try {
+				logger.debug(`Running Storage.url.get with ${id}`)
+				return this.driver.url.get(id)
+			} catch (err) {
+				logger.error(`Storage.url.get failed: ${err}`)
```

Comment:
Did you check that these logs actually show the error well? If they do, just ignore this comment.